src/app/models/gtypes/album_model.rs

```diff
@@ -141,6 +168,7 @@ mod imp {
             match pspec.name() {
                 "album" => self.album.borrow().to_value(),
                 "artist" => self.artist.borrow().to_value(),
+                "year" => self.year.borrow().unwrap_or(0).to_value(),
```

Comment:
I don't know how easy it would be because of the borrow call (always a bit tricky sadly) but unwrapping should ideally not be needed.

you might have to toy around with [Ref::map](https://doc.rust-lang.org/std/cell/struct.Ref.html#method.map)